stock_vertical_lift/models/vertical_lift_operation_base.py

```diff
@@ -287,6 +287,12 @@ def process_current(self):
         """
         raise NotImplementedError
 
+    def button_skip(self):
```

Comment:
Is the skip button going to be implemented for Put/Inventory? If not, the `button_skip` button better has to be in vertical_lift_operation_pick.py only

Reply:
OK, it's now a method of the child class, ```vertical.lift.operation.pick```.